docs/.vitepress/theme/components/VersionInfo.vue

```diff
@@ -0,0 +1,15 @@
+<script setup lang="ts">
+import { version } from '../../../package.json'
+</script>
+
+<template>
+  <div class="flex justify-between items-center">
+    <h1 id="kit-cli-reference" tabindex="-1">
+      Kit CLI Reference
```

Comment:
All these changes are good to me, but in the future, and for a bigger reusability, we want this line to be a `<slot />` so we can pass any title content that we want, and therefor, reuse it from anyplace.

Reply:
I was going to do something like that, but I couldn't figure out a good way to parameterize the id/anchor bits -- if you've got tips I'm happy to fix it :+1:

TBH I copied the anchor from what Vite autogens for a markdown `#` 🙂